### PR TITLE
Update index image

### PR DESCRIPTION
**Pull request by JM-GA**

### CODE REVIEW

File: index/constraints.txt [hunk 1]
```diff
@@ -151,11 +154,11 @@ numpy==1.23.2
     #   rasterio
     #   snuggs
     #   xarray
-odc-apps-cloud==0.2.2.dev3382
+odc-apps-cloud==0.2.1
```
Comment (pindge) [inline index/constraints.txt:157]: has this not been updated?

**Review by pindge — CHANGES_REQUESTED** [review]

File: index/constraints.txt (hunk 1 above)
Comment (JM-GA) [reply index/constraints.txt:157]: @pindge 0.2.1 is latest non dev version I believe
https://pypi.org/project/odc-apps-cloud/#history

Comment (pindge) [reply index/constraints.txt:157]: no, those are not build from pypi, they are built from https://packages.dea.ga.gov.au/odc-apps-cloud/

File: index/constraints.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.10
+# This file is autogenerated by pip-compile with python 3.8
 # To update, run:
 #
 #    pip-compile --extra-index-url=https://packages.dea.ga.gov.au/ --output-file=constraints.txt requirements.txt
```
Comment (pindge) [inline index/constraints.txt:5]: notice this `extra-index-url`

**Review by pindge — CHANGES_REQUESTED** [review]

File: index/constraints.txt (hunk 1 above)
Comment (jeremyh) [reply index/constraints.txt:157]: Strange. The extra package repo was included (the "To update this," line in the new `constraints.txt` would be different if it wasn't). 

pip-compile doesn't include pre-releases of libraries by default, so maybe it was run last time with `--pre`?

Comment (pindge) [reply index/constraints.txt]: it depends on which code base you need, lot of the time, the code we need hasn't been released, the thing is, we need to learn how to do a release from odc-tools first.

Comment (jeremyh) [reply index/constraints.txt]: That makes sense.

In that case, we probably want to make it explicit by adding a min version in `requirements.txt` @JM-GA.

Change the odc-apps-cloud line to `odc-apps-cloud>=0.2.2.dev3382`, and then recompile.

**Review by pindge — APPROVED** [review]

**Pull request merged**